src/components/PageThemes.jsx

```diff
@@ -0,0 +1,21 @@
+import { useOutletContext } from "react-router-dom";
+import Button from "./Button"
+
+const HomePage = () => {
+  const { setThemeColor } = useOutletContext();
+  const { theme } = useOutletContext();
+
+  return (
+    <>
+      <h1 style={{color: theme.h1Color}} className="text-center no-pad">Page Themes</h1>
+      
+      <div className='row d-flex justify-content-center'>
+          <Button style={ theme.btnColor } label="Default" onClick={() => {setThemeColor(1)}} />
```

Comment:
Ideally, you would consume your context at the component level so you wouldn't have to pass the prop each time.